tests/fast-integration/kyma-environment-broker/helpers.js

```diff
@@ -102,11 +102,11 @@ async function ensureOperationSucceeded(keb, kcp, instanceID, operationID, timeo
 
   if (res.state !== 'succeeded') {
     const runtimeStatus = await kcp.getRuntimeStatusOperations(instanceID);
-    throw new Error(`operation didn't succeed in time: ${JSON.stringify(res, null, `\t`)}
+    throw new Error(`operation didn't succeed in time: ${JSON.stringify(res, null, `\t`)}\n
```

Comment:
nit: is the newline character `\n` at the end desired? I think the fact this is a multiline string will print it on multiple lines.

Reply:
in line `106` below, the `runtimeStatus` is also printed - that's why the newline character is there